starfyre/component.py

```diff
@@ -25,3 +25,6 @@ def render(self):
     @property
     def is_text_component(self):
         return self.tag == "TEXT_NODE"
+    
+    def __repr__(self):
+        return f"<{self.tag}> {self.data} {self.children}"
```

Comment:
One final nit - can we print it like this 

`f"<{self.tag}> {self.data} {self.children} </{self.tag}>`

Reply:
Yes, I will update it.